code/bit_manipulation/src/sorted subsequences/sorted subsequences.cpp

```diff
@@ -0,0 +1,24 @@
+#include<bits/stdc++.h>
+
+using namespace std;
```

Comment:
Remove this.

Reply:
it willl make program more complex

Reply:
@aryangulati But it is bad practice to keep it. Googling it will convince you of this.